edge_model_serving/tflite_serving/pyproject.toml

```diff
@@ -32,8 +32,8 @@ dev = [
 linux = [
     "tflite-runtime==2.7.0"
 ]
-mac = [
-    "tflite-runtime==2.5.0"
+macos = [
```

Comment:
to be remove and add link to https://www.tensorflow.org/lite/guide/build_cmake_pip, not tested